src/server/handlers/github-auth.js

```diff
@@ -0,0 +1,59 @@
+import request from 'request';
+import logging from '../logging';
+
+const AUTH_SCOPE = 'admin:repo_hook';
```

Comment:
Not sure if private repos are in scope. Even if we would get access to them here, still LP would need to be authorised separately to access them to build.